### ISSUE TITLE
Input fields for HeatExchanger:Desiccant:BalancedFlow:PerformanceDataType1 object claim to be "Autosizable", but aren't

### ISSUE THREAD

**Issue by aaron-boranian** [NREL/EnergyPlus#5385]
When the IDD file describes input fields for the HeatExchanger:Desiccant:BalancedFlow:PerformanceDataType1 object, it claims that 'Nominal Air Flow Rate' and 'Nominal Air Face Velocity' are autosizable. 

```
HeatExchanger:Desiccant:BalancedFlow:PerformanceDataType1,
       .... 
  N1,  \field Nominal Air Flow Rate
       \note Air flow rate at nominal conditions (assumed to be the same for both sides
       \note of the heat exchanger).
       \type real
       \units m3/s
       \minimum> 0.0
       \autosizable
  N2 , \field Nominal Air Face Velocity
       \required-field
       \type real
       \units m/s
       \minimum> 0.0
       \maximum 6.0
       \autosizable
```

However, if "Autosize" is entered for either input field in an IDF file, EnergyPlus crashes with the following severe error message:

```
 ** Severe  ** HeatExchanger:Desiccant:BalancedFlow:PerformanceDataType1 "CAV DESICCANT HX PERF CURVE"
 **   ~~~   ** Nominal air flow rate must be greater than zero.
 **   ~~~   ** ... value entered = -99999.000000
```

Even though this is how the input fields are entered in the IDF:

```
    HeatExchanger:Desiccant:BalancedFlow:PerformanceDataType1,
      CAV Desiccant HX Perf Curve,              !- Name
      Autosize,                    !- Nominal Air Flow Rate {m3/s}
      3.25,                    !- Nominal Air Face Velocity {m/s}
      ... 
```

It seems like somewhere, somehow E+ is converting the "Autosize" input into the value -99999. I'm using V8.4 and you can find the original IDF file [HERE](https://drive.google.com/file/d/0B_SyIwm6jC9MTS12aDhRdVNZVDQ/view?usp=sharing). I copied the Desiccant HX Performance Data object from the DesiccantDehumidifierWithCompanionCoil.idf EnergyPlus example file, and am using the object in combination with the HeatExchanger:Desiccant:BalancedFlow and Dehumidifier:Desiccant:System objects.

**Comment by rraustad** [comment]
Sure enough, autosize is not handled for the HeatExchanger:Desiccant:BalancedFlow:PerformanceDataType1 object.

**Comment by mjwitte** [comment]
Closed via #5743
